api/ruby/find-inactive-members/README.md

```diff
@@ -0,0 +1,46 @@
+# Find Inactive Organization Members
+> a utility to find, and optionally remove, inactive organization members
+
+This utility finds users inactive since a configured date, writes those users to a file `inactive_users.csv`, and optionally removes them from the organization
+
+## Installation
+
+### Clone this repository
+
+```shell
+git clone https://github.com/github/platform-samples.git
+cd api/ruby/find-inactive-members
+```
+
+### Install dependencies
+
+```shell
+gem install octokit
+```
+
+### Configure Octokit
+
+```shell
+export OCTOKIT_API_ENDPOINT="https://github.example.com/api/v3" # Default: "https://api.github.com"
```

Comment:
`GITHUB_API_ENDPOINT`?

Reply:
The docs are right. I'll update the code to be accurate.